core/src/main/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotService.scala

```diff
@@ -87,13 +91,24 @@ class SnapshotService(protected val userInfo: UserInfo, val dataSource: SlickDat
     }
   }
 
-  def enumerateSnapshots(workspaceName: WorkspaceName, offset: Int, limit: Int): Future[ResourceList] = {
+  //AS-787 - rework the data so that it's in the same place in the JSON with a list and get snapshot responses
+  def massageSnapshots(references: ResourceList): Map[String, Seq[DataRepoSnapshotResource]] = {
+    val snapshots = references.getResources.asScala.map { r =>
+      val massaged = new DataRepoSnapshotResource
+      massaged.setAttributes(r.getResourceAttributes.getGcpDataRepoSnapshot)
+      massaged.setMetadata(r.getMetadata)
+      massaged
+    }
+    Map(SnapshotService.GCP_SNAPSHOTS_KEY -> snapshots)
+  }
```

Comment:
best practice is to make a new case class with a "gcpDataRepoSnapshots" member, then return an instance of that class instead of a `Map`. It's a bit more overhead but it keeps everything type-safe, and other code such as the unit tests doesn't have to be searching through a map for a well-known key.